.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+# This GitHub action runs your tests for each commit push and/or PR. Optionally
+# you can turn it on using a cron schedule for regular testing.
+#
+name: Tests
+on:
+  pull_request:
+    paths-ignore:
+      - "README.md"
+  push:
+    paths-ignore:
+      - "README.md"
+
+jobs:
```

Comment:
We don't need 2 workflows to run test, this can be a single workflow that lint at one step then test. there is no need to have 2 workflows that do exactly the same thing